server/src/modules/enquiry/__tests__/enquiry.service.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { EnquiryService } from '../enquiry.service'
+import { MailService } from '../../mail/mail.service'
+import { Enquiry } from '../../../types/mail-type'
+import { Sequelize } from 'sequelize'
+import {
+  defineAgency,
+  defineTag,
+  defineUserAndPermission,
+} from '../../../models'
+import minimatch from 'minimatch'
+
+const mockedSequelize = new Sequelize({
+  dialect: 'sqlite',
+  username: 'username',
+  logging: false,
+})
```

Comment:
`mockedSequelize` isn't entirely accurate; you are using an actual sequelize instance, just that it's using sqlite